integrationtest/pom.xml

```diff
@@ -155,26 +155,10 @@
             <groupId>${project.groupId}</groupId>
             <artifactId>hibernate-ogm-ignite</artifactId>
             <scope>provided</scope>
+            <version>${project.version}</version>
```

Comment:
@schernolyas There is no need to add the version to these dependencies, it is inherited from the parent